ratis-server/src/main/java/org/apache/ratis/server/impl/FollowerState.java

```diff
@@ -88,6 +93,20 @@ void stopRunning() {
     this.isRunning = false;
   }
 
+  boolean lostMajorityHeartbeats() {
```

Comment:
how about rename to `isTimeoutSinceLostMajorityHeartbeats` ?

Reply:
I got that you wanted to invert the boolean.  However, "isTimeoutSinceLostMajorityHeartbeats" is hard to understand.  So, let's keep "lostMajorityHeartbeats"?  Or I can change it to ""lostMajorityHeartbeatsRecently".

Reply:
Okay, `lostMajorityHeartbeatsRecently` is better.

Reply:
Done.